### PR TITLE
update contributing.md and style.md

### PR DESCRIPTION
**Pull request by emilybrooks**
With PlayState being renamed I wanted to revise the style guide. There's also a few things that I thought would be helpful to clarify, such as leaving things unnamed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,9 +9,28 @@ Thank you for your interest in the Animal Forest decompilation. This is a brief
 
 Most discussions happen in the `#af-decomp` channel on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started.
 
-## Preparing a Pull Request
+## Tips for Successful Pull Requests
 
-You should work on a branch on your fork separate from your copy of main: it is always useful to have a clean main branch around if you need to fix something.
+- Don't bundle together multiple files worth of decompiled code into a single pr. Large prs require a lot of effort to review, and make things difficult for everyone.
+- You don't necessarily have to decompile a whole file, you can just decompile a couple of functions in a single pr if that suits your workflow better.
```
Comment (AngheloAlf) [inline CONTRIBUTING.md:15]: The rest of the file says "PR" in uppercase, so I think it should be changed for consistency
```suggestion
- Don't bundle together multiple files worth of decompiled code into a single PR. Large PRs require a lot of effort to review, and make things difficult for everyone.
- You don't necessarily have to decompile a whole file, you can just decompile a couple of functions in a single PR if that suits your workflow better.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,9 +9,28 @@ Thank you for your interest in the Animal Forest decompilation. This is a brief
...
+- Always merge main instead of rebasing. Rebasing changes how commits are shown in pull requests, making changes difficult to follow.
```
Comment (AngheloAlf) [inline CONTRIBUTING.md:16]: It should be noted that this is only an issue when the PR is already open. Before opening a PR is fine to rebase a branch

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,9 +9,28 @@ Thank you for your interest in the Animal Forest decompilation. This is a brief
...
+- Review the [Style Guide](docs/STYLE.md) and see if there's anything you can preemptively fix before code reviews.
+
+## Making a Pull Request
+
+You should work on a branch on your fork separate from your copy of main. You can create branches from `upstream/main` so you don't have to sync your main branch every time.
```
Comment (AngheloAlf) [inline CONTRIBUTING.md:21]:
```suggestion
You should work on a branch on your fork separate from your copy of `main`. You can create branches from `upstream/main` so you don't have to sync your main branch every time.
```

File: docs/STYLE.md [hunk 4]
```diff
@@ -165,20 +170,25 @@ We use comments for:
...
 This section mostly applies to actors.
 
 ### Functions
 
-All functions should go in the main C file in the same order as the assembly (the latter is required to match anyway). (We may make exceptions for particularly large files with a particular organisational structure, but we ask that you check on Discord first before doing this)
+All functions should go in the main C file in the same order as the assembly (the latter is required to match anyway). (We may make exceptions for particularly large files with a particular organizational structure, but we ask that you check on Discord first before doing this)
+
+### Prototypes
+
+Only put prototypes that are necessary. They should be placed in the main C file, unless they are used by something outside of the file, where they should be placed in the header.
```
Comment (AngheloAlf) [inline docs/STYLE.md:185]: Do you mean the .h file instead of .c file here?

**Review by AngheloAlf — COMMENTED** [review]
Thanks for revisiting this files

File: docs/STYLE.md (hunk 4 above)
Comment (emilybrooks) [reply docs/STYLE.md:185]: I meant for actors, I guess I should make a separate note for code files to avoid confusion.

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**